src/daemon/utils_cache.h

```diff
@@ -48,6 +48,9 @@ int uc_set_state (const data_set_t *ds, const value_list_t *vl, int state);
 int uc_get_hits (const data_set_t *ds, const value_list_t *vl);
 int uc_set_hits (const data_set_t *ds, const value_list_t *vl, int hits);
 int uc_inc_hits (const data_set_t *ds, const value_list_t *vl, int step);
+int uc_get_persist_frequency (const data_set_t *ds, const value_list_t *vl);
```

Comment:
Could we use `uc_meta_data_get_signed_int()` to avoid this highly specific interface?